pkg/login/social/azuread_oauth_test.go

```diff
@@ -249,6 +254,54 @@ func TestSocialAzureAD_UserInfo(t *testing.T) {
 				Groups:  []string{"foo"},
 			},
 		},
+		{
+			name: "Fetch groups when ClaimsNames and ClaimsSources is set",
+			fields: fields{
+				SocialBase: newSocialBase("azuread", &oauth2.Config{}, &OAuthInfo{}),
+			},
+			claims: &azureClaims{
+				ID:                "1",
+				Name:              "test",
+				PreferredUsername: "test",
+				Email:             "test@test.com",
+				Roles:             []string{"Viewer"},
+				ClaimNames:        claimNames{Groups: "src1"},
```

Comment:
```suggestion
				ClaimNames:        claimNames{Groups: "src1"},
				ClaimSources:      nil, // set by the test
```

Maybe this is silly, wdyt?